### PR TITLE
Add demo links to readme

### PR DESCRIPTION
**Pull request by yakhinvadim**
## Description

- Added link to Vue.js project in "Is Search UI only for React?" question in FAQ
- Added "Can I use my own styles?" question in FAQ with a link to Constance's demo

### CODE REVIEW

**Comment by yakhinvadim** [comment]
@constancecchen Could you, please, put a link to your project on the project's github page?
https://github.com/constancecchen/seattle-indies-expo-search
https://constance.dev/seattle-indies-expo-search/

File: README.md [hunk 1]
```diff
@@ -265,7 +265,14 @@ Nope. Search UI is "headless".
 
 You can write support for it into any JavaScript framework. You can even use vanilla JavaScript.
 
-[Read the Headless Core Guide](./ADVANCED.md#headless-core) for more information.
+[Read the Headless Core Guide](./ADVANCED.md#headless-core) for more information, or check
+out the [Vue.js Example](https://github.com/yakhinvadim/vue-search-ui-demo).
```
Comment (goodroot) [inline README.md]: We might want to re-create this as an Elastic repository.

File: README.md [hunk 2]
```diff
@@ -265,7 +265,14 @@ Nope. Search UI is "headless".
...
+
+### Can I use my own styles?
+
+You can!
+
+Read the [Custom Styles and Layout Guide](./ADVANCED.md#custom-styles-and-layout) to learn more, or check out the [Seattle Indies Expo Demo](https://github.com/constancecchen/seattle-indies-expo-search).
```
Comment (goodroot) [inline README.md]: Like-wise: consider re-creating this as an Elastic repository.

File: README.md (hunk 1 above)
Comment (constancecchen) [reply README.md]: Just curious - any specific reason for the manual newline vs. just letting IDEs wordwrap naturally?

Comment (yakhinvadim) [reply README.md]: Oops, no reason :) Thx for noticing.

**Comment by yakhinvadim** [comment]
Thank you all! And a great suggestion to move repos to Elastic org.

**Comment by JasonStoltz** [comment]
I like that the demo links are tied to those two FAQs.

**Pull request merged**